dozer-api/src/generator/oapi/utils.rs

```diff
@@ -103,10 +103,12 @@ pub fn convert_cache_to_oapi_schema(
 /// Should be consistent with `field_to_json_value`.
 fn convert_cache_type_to_schema_type(field_type: dozer_types::types::FieldType) -> Type {
     match field_type {
-        FieldType::UInt | FieldType::Int => Type::Integer(IntegerType {
-            format: VariantOrUnknownOrEmpty::Item(IntegerFormat::Int64),
-            ..Default::default()
-        }),
+        FieldType::UInt | FieldType::U128 | FieldType::Int | FieldType::I128 => {
```

Comment:
`U128` and `I128` should be `Type::String`.

Reply:
right